workflow/applications/applications.py

```diff
@@ -71,6 +71,7 @@ def __init__(self, conf: Configuration) -> None:
         self.do_goes = _base.get('DO_GOES', False)
         self.do_mos = _base.get('DO_MOS', False)
         self.do_extractvars = _base.get('DO_EXTRACTVARS', False)
+        self.do_repair_replay = _base.get('DO_REPAIR_REPLAY', False)
```

Comment:
@HongGuan-NOAA When you merge `develop`, I believe this will need to be changed. 
```suggestion
        self.do_repair_replay = base.get('DO_REPAIR_REPLAY', False)
```

Reply:
@HongGuan-NOAA We might not need to worry about this in this PR because your target is `feature/gefs_reforecast`, not `develop`.